src/test_l1_dsDisplay.c

```diff
@@ -149,30 +145,30 @@ void test_l1_dsDisplay_positive_dsDisplayInit(void) {
     UT_LOG("\n Out %s\n", __FUNCTION__);
 }
 
-
 /**
  * @brief Ensure dsDisplayInit() returns correct error codes during negative scenarios
- * 
+ *
  * **Test Group ID:** Basic: 01@n
  * **Test Case ID:** 002@n
- * 
+ *
  * **Pre-Conditions:**@n
- * 
+ *
  * **Dependencies:** None@n
  * **User Interaction:** None
- * 
+ *
```

Comment:
There is a problem here for source devices, if source ports are zero and non of the tests runs for source device except init and term. Its better to add 1 port in yaml and type should be HDMI for source devices

Reply:
addressed in PR 
https://github.com/rdkcentral/rdk-halif-test-device_settings/pull/127